pulsar-io/kafka-connect-adaptor/src/main/java/org/apache/pulsar/io/kafka/connect/PulsarKafkaWorkerConfig.java

```diff
@@ -53,10 +59,14 @@
             .define(PULSAR_SERVICE_URL_CONFIG,
                 Type.STRING,
                 Importance.HIGH,
-                PULSAR_SERVICE_URL_CONFIG_DOC);
+                PULSAR_SERVICE_URL_CONFIG_DOC)
+            .define(TOPIC_NAMESPACE_CONFIG,
+                Type.STRING,
+                "public/default/",
```

Comment:
I would suggest removing '/' here. since `<tenant>/<namespace>` is the pattern we used in elsewhere in pulsar project.